servers_v7.json

```diff
@@ -1,5 +1,9 @@
 [
   {
     "address": "mindustry.us.to"
+  },
+  {
+    "name": "Omega"
```

Comment:
```suggestion
    "name": "Omega",
```

Reply:
also, this same ip is also (still) on the bleeding edge list 🤔

Reply:
k thx

Reply:
yeah, gona remove it from there too